variables.tf

```diff
@@ -11,16 +11,19 @@ variable "certificate_name" {
 
 variable "create_wildcard" {
```

Comment:
Suggest you just add the `type = string` to `hosted_zone_name` and `certificate_name` as well